code/modules/reagents/chemistry/reagents/atmos_gas_reagents.dm

```diff
@@ -0,0 +1,158 @@
+
+/datum/reagent/hypernoblium/on_mob_end_metabolize(mob/living/breather)
+	if(isplasmaman(breather))
+		REMOVE_TRAIT(breather, TRAIT_NOFIRE, type)
+	return ..()
+
+/datum/reagent/healium
+	name = "Healium"
+	description = "A powerful sleeping agent with healing properties"
+	reagent_state = GAS
+	metabolization_rate = REAGENTS_METABOLISM * 0.5
+	color = "90560B"
+	taste_description = "rubbery"
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED|REAGENT_NO_RANDOM_RECIPE
+
+/datum/reagent/healium/on_mob_metabolize(mob/living/breather)
+	. = ..()
+	breather.PermaSleeping()
+
+/datum/reagent/healium/on_mob_end_metabolize(mob/living/breather)
+	breather.SetSleeping(10)
+	return ..()
+
+/datum/reagent/healium/on_mob_life(mob/living/breather, delta_time, times_fired)
+	breather.adjustFireLoss(-2 * REM * delta_time, FALSE)
+	breather.adjustToxLoss(-5 * REM * delta_time, FALSE)
+	breather.adjustBruteLoss(-2 * REM * delta_time, FALSE)
+	..()
+	return TRUE
+
+/datum/reagent/halon
+	name = "Halon"
+	description = "A fire suppression gas that removes oxygen and cools down the area"
+	reagent_state = GAS
+	metabolization_rate = REAGENTS_METABOLISM * 0.5
+	color = "90560B"
+	taste_description = "minty"
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED|REAGENT_NO_RANDOM_RECIPE
+
+/datum/reagent/halon/on_mob_metabolize(mob/living/breather)
+	. = ..()
+	breather.add_movespeed_modifier(/datum/movespeed_modifier/reagent/halon)
+	ADD_TRAIT(breather, TRAIT_RESISTHEAT, type)
+
+/datum/reagent/halon/on_mob_end_metabolize(mob/living/breather)
+	breather.remove_movespeed_modifier(/datum/movespeed_modifier/reagent/halon)
+	REMOVE_TRAIT(breather, TRAIT_RESISTHEAT, type)
+	return ..()
+
+/datum/reagent/zauker
+	name = "Zauker"
+	description = "An unstable gas that is toxic to all living beings."
+	reagent_state = GAS
+	metabolization_rate = REAGENTS_METABOLISM * 0.5
+	color = "90560B"
+	taste_description = "bitter"
+	chemical_flags = REAGENT_NO_RANDOM_RECIPE
+
+/datum/reagent/zauker/on_mob_life(mob/living/breather, delta_time, times_fired)
+	breather.adjustBruteLoss(6 * REM * delta_time, FALSE)
+	breather.adjustOxyLoss(1 * REM * delta_time, FALSE)
+	breather.adjustFireLoss(2 * REM * delta_time, FALSE)
+	breather.adjustToxLoss(2 * REM * delta_time, FALSE)
+	..()
+	return TRUE
+
+/datum/reagent/pluoxium
+	name = "Pluoxium"
+	description = "A gas that is eight times more efficient than O2 at lung diffusion with organ healing properties on sleeping patients."
+	reagent_state = GAS
+	metabolization_rate = REAGENTS_METABOLISM * 0.5
+	color = "#808080"
+	taste_description = "irradiated air"
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED|REAGENT_NO_RANDOM_RECIPE
+
+/datum/reagent/pluoxium/on_mob_life(mob/living/carbon/breather, delta_time, times_fired)
+	if(!HAS_TRAIT(breather, TRAIT_KNOCKEDOUT))
+		..()
+		return ..()
+
+	for(var/organ as anything in breather.internal_organs)
+		var/obj/item/organ/organ_being_healed = organ
+		organ_being_healed.applyOrganDamage(-0.5)
```

Comment:
no no no no no delta time please no

Reply:
Well I put the REM/Delta time as asked, would be nice to know how and why this is an issue and if it should be reverted.

Reply:
> I put the REM/Delta time as asked, would be nice to know how/why this is an issue and if it should be reverted.

Delta time does not belong on chems and shit. Why? Damaging chems and consistency.

Damaging chems shouldn't have delta time because if the server hits a fat lag spike, congrats, you're dead and it was out of your control. For consistency reasons, this means no chems should have delta time. Delta time bad.

Reply:
just adds to the espionage opportunity